addons/new-osio-addon/src/main/java/io/fabric8/launcher/osio/steps/OpenShiftSteps.java

```diff
@@ -43,4 +105,34 @@ public void createBuildConfig(OsioProjectile projectile) {
         annotations.put(Annotations.JENKINS_DISABLE_SYNC_CREATE_ON, "jenkins");
         return annotations;
     }
+
+    private String getSpaceIdFromSpacePath(String spacePath) {
+        return spacePath.substring(1);
```

Comment:
Why is this method doing a substring?

Reply:
because spacePath looks like "/my-space" and we need only the my-space part